recipe/pre-unlink.sh

```diff
@@ -1,4 +1,2 @@
-{
-  "${PREFIX}/bin/jupyter-nbextension" disable --sys-prefix --py nbgrader
-  "${PREFIX}/bin/jupyter-serverextension" disable --sys-prefix --py nbgrader
-} >>"$PREFIX/.messages.txt" 2>&1
+"${PREFIX}/bin/jupyter-nbextension" uninstall nbgrader --py --sys-prefix >> "${PREFIX}/.messages.txt" 2>&1
+"${PREFIX}/bin/jupyter-serverextension" disable nbgrader --py --sys-prefix >> "${PREFIX}/.messages.txt" 2>&1
```

Comment:
I am not sure we need to wrap the commands in `{}` there... But I can revert this change if there is a reason to do it.